torch/csrc/jit/fuser/cpu/fused_kernel.cpp

```diff
@@ -229,7 +231,7 @@ static std::string getArchFlags() {
 static const std::string arch_flags = getArchFlags();
 static const std::string compile_string =
     "cd /D \"" + temp_dir + "\" && "
-    "${cxx} /nologo /MD /Ox " + arch_flags + " /LD /EHsc "
+    "${cxx} /nologo /MD /O2 " + arch_flags + " /LD /EHsc "
```

Comment:
Although it is quite unbelievable, actually `/O2` is preferred over `/Ox` in MSVC if user want performance-optimized binary. See https://stackoverflow.com/questions/29633566/visual-studio-2013-optimization-flags-o2-vs-ox for more details.